README.md

```diff
@@ -10,30 +10,22 @@ status: draft
 
 # Introduction
 
-In this tutorial, we are going to build a simple web application using [AdonisJS](https://adonisjs.com/) and integrate it with [Materialize](https://materialize.com).
+In this tutorial, we are going to build a web application using [AdonisJS](https://adonisjs.com/) and integrate it with [Materialize](https://materialize.com) to .
 
-# Prerequisits
+# Prerequisites
 
-You would need to have the following things installed before you begin:
+You need to have the following things installed before getting started:
 
-* [Install Docker and Docker Compose](https://www.digitalocean.com/community/tutorials/how-to-install-and-use-docker-compose-on-ubuntu-20-04)
-* [Install Node.js](https://devdojo.com/ruanbekker/install-nodejs-on-linux-using-nvm)
```

Comment:
It'd be nice to point to more generic resources that are less tied to "just Linux".